Destiny.ScrimTracker.Logic/Models/GuardianElo.cs

```diff
@@ -0,0 +1,10 @@
+namespace Destiny.ScrimTracker.Logic.Models
+{
+    public class GuardianElo
+    {
+        public int MatchId { get; set; }
```

Comment:
Do you think this should link to the `Match` or `MatchResult` or both? I'm honestly not sure what the best approach here is...

Reply:
I think linking to the match might be better? Since you can get to individual match result from there that seems to be enough information